osu.Game.Tests/Visual/Gameplay/TestSceneSpectatorPlayback.cs

```diff
@@ -354,7 +356,7 @@ public enum TestAction
         internal class TestReplayRecorder : ReplayRecorder<TestAction>
         {
             public TestReplayRecorder()
-                : base(new Score())
+                : base(new Score { ScoreInfo = { Beatmap = new TestBeatmap(new OsuRuleset().RulesetInfo).BeatmapInfo } })
```

Comment:
```suggestion
                : base(new Score { ScoreInfo = { Beatmap = new BeatmapInfo() } })
```

Above can also use this if deemed more appropriate, just feels better as it's not actually meaningful for this test to make a full `TestBeatmap`.